src/users/users.ts

```diff
@@ -217,4 +219,20 @@ export class Users {
 
     return deserializeUser(data);
   }
+
+  async updateUser({
+    userId,
+    lastName,
+    firstName,
+  }: UpdateUserOptions): Promise<User> {
+    const { data } = await this.workos.put<
+      UserResponse,
+      SerializedUpdateUserOptions
+    >(`/users/${userId}`, {
```

Comment:
Should we have a dedicated serialization function like we do here? https://github.com/workos/workos-node/blob/c99d90819c6241d3612f18f42aaf62195f877a58/src/organizations/organizations.ts#L40